run.py

```diff
@@ -810,7 +824,22 @@ def run(self, outdir):
 
                 p = subprocess.Popen(
                     args, shell=shell, cwd=self.directory, env=env,
-                    stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                    stdout=subprocess.PIPE, stderr=subprocess.PIPE, start_new_session=True)
+
+                if self.suricata_config.interface != None:
+                    argsl = ["tcpreplay", "-i", self.suricata_config.interface]
+                    pcap = self.pcap_name()
+                    if pcap == None:
+                        raise TestError("live test needs a pcap")
+                    argsl.append(pcap)
+
+                    stdout.write(self.wait_suricata_start(p))
+                    pl = subprocess.check_call(
+                        argsl, shell=shell, cwd=self.directory, env=env,
+                        stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                    # TODO how to know Suricata completed ?
```

Comment:
any ideas here ? Besides adding a unix-socket command ? or wait for a stats event with the number of packets